### DISCUSSION TITLE
Date picker for running queries tab

### DISCUSSION THREAD

**Discussion by alinselicean** [trimble-oss/dba-dash#377] [Ideas]
Hi - there may be situations when one might need to dig deeper into the historical data, like few days old. For instance, now I have to navigate all the back to the 4th of the month (see screenshot below). If there's another way of achieving this, I am all for it :) 

But I think a calendar picker will greatly help getting to that data much faster, in addition to what is now available for finer navigation control.

Thanks and thanks for sharing this great tool.

![image](https://user-images.githubusercontent.com/20295322/195266342-ef9d60c5-ca35-4e02-8dd9-6cc4b82a9b8c.png)

**Comment by DavidWiseman** [comment]
Hi,

The running queries tab will display the snapshots available for the selected date range.  If you want to go back to the 4th, you can select just that day from the menu:
![image](https://user-images.githubusercontent.com/33157668/195272104-7d232184-a9bf-43f7-b2f1-40bb8103724f.png)

If you want to narrow things down a bit further, you can select Custom.  You can then pick a custom date range - between 11am and 12pm on the 4th for example.

![image](https://user-images.githubusercontent.com/33157668/195272505-11887bba-8011-4355-941b-f95fdbdd11f5.png)

Hope this helps,

David

**Comment by alinselicean** [comment]
Thanks. I thought I might be missing something or doing it wrong :)
